safedelete/queryset.py

```diff
@@ -90,10 +91,12 @@ def _filter_visibility(self):
         This is because QuerySet._fetch_all cannot work with a clone.
         """
         force_visibility = getattr(self, '_safedelete_force_visibility', None)
+        filter_applied = getattr(self, '_safedelete_filter_applied', False)
```

Comment:
Can't we use `self._safedelete_filter_applied` instead of `getattr(self, '_safedelete_filter_applied', False)`? The variable is always going to be defined if I am not mistaken?